src/Workspaces/Core/Portable/Workspace/Solution/RecoverableTextAndVersion.cs

```diff
@@ -138,7 +138,7 @@ public async ValueTask<VersionStamp> GetVersionAsync(LoadTextOptions options, Ca
             return recoverableText.Version;
         }
 
-        private sealed class RecoverableText : WeaklyCachedRecoverableValueSource<SourceText>
+        private sealed partial class RecoverableText : ValueSource<SourceText>
```

Comment:
Change looks good to me. Could you please combine the partial definitions of RecoverableText into one? I'm having trouble keeping track of partial type inside another partial type :)

Reply:
Sure!  I wanted to keep them separately initially so we could see that nothing else changed.  I'll merge in upcoming PR.